tomviz/OperatorPython.cxx

```diff
@@ -399,13 +399,21 @@ Operator* OperatorPython::clone() const
 
 bool OperatorPython::serialize(pugi::xml_node& ns) const
 {
+  ns.append_attribute("json_description")
+    .set_value(this->JSONDescription().toLatin1().data());
   ns.append_attribute("label").set_value(this->label().toLatin1().data());
   ns.append_attribute("script").set_value(this->script().toLatin1().data());
+  if (this->hasChildDataSource() && this->childDataSource()) {
+    pugi::xml_node childNode = ns.append_child("DataSource");
+    this->childDataSource()->serialize(childNode);
+  }
+
   return true;
 }
 
 bool OperatorPython::deserialize(const pugi::xml_node& ns)
 {
+  this->setJSONDescription(ns.attribute("json_description").as_string());
```

Comment:
I was worried this would break all older state files, but after testing it, it seems that it doesn't.  I guess it just defaults to empty string and that is handled fine elsewhere.